cider/exec/nextgen/jitlib/llvmjit/LLVMJITFunction.cpp

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2022 Intel Corporation.
+ * Copyright (c) OmniSci, Inc. and its affiliates.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+#include "exec/nextgen/jitlib/llvmjit/LLVMJITFunction.h"
+
+#include <llvm/IR/Function.h>
+#include <llvm/IR/Verifier.h>
+#include <llvm/Support/raw_os_ostream.h>
+
+#include "exec/nextgen/jitlib/llvmjit/LLVMJITModule.h"
+#include "util/Logger.h"
+
+namespace jitlib {
+LLVMJITFunction::LLVMJITFunction(const JITFunctionDescriptor& descriptor,
+                                 LLVMJITModule& module,
+                                 llvm::Function& func)
+    : JITFunction<LLVMJITFunction>(descriptor)
+    , module_(module)
+    , func_(func)
+    , ir_builder_(nullptr) {
+  auto local_variable_block =
+      llvm::BasicBlock::Create(getLLVMContext(), ".Local_Vars", &func_);
+  auto entry_block = llvm::BasicBlock::Create(getLLVMContext(), ".Entry", &func_);
```

Comment:
seems a little weird to name the second block ".Entry", llvm always assume the fisrt block is entry. let's name it ".Start" or any other?